Gemfile

```diff
@@ -2,6 +2,9 @@ source 'https://rubygems.org'
 
 ruby '2.2.2'
 
+# force Bundler to use HTTPS for github repos
```

Comment:
Yep, copied and pasted from another repo (with slightly different constraints) in haste :(